config/pm2_servers/dev.json

```diff
@@ -31,6 +31,23 @@
             "instances": 1,
             "name": "bpm_frontend-dev",
             "script": "./node_modules/.bin/webpack-dev-server"
-        }
-    ]
+        }, {
+            "args": [
+                "--config=./src/admin/webpack.config.babel.js",
+                "--content-base=build/admin",
+                "--devtool=cheap-module-inline-source-map",
```

Comment:
I haven't been able to move everything in the webpack config. `hot` and `port` don't work